db/models/__init__.py

```diff
@@ -105,37 +107,34 @@ def vlan_with_mask(cls, v):
 class FlowSubDoc(BaseModel):
     """Flow DB SubDocument Model."""
 
-    table_id = 0
-    owner: Optional[str]
-    table_group = "base"
-    priority = 0x8000
-    cookie: Decimal128 = Decimal128("0")
-    idle_timeout = 0
-    hard_timeout = 0
-    match: Optional[MatchSubDoc]
-    actions: Optional[List[dict]]
-    instructions: Optional[List[dict]]
-
-    class Config:
-        """Config."""
-
-        arbitrary_types_allowed = True
-
-    @validator("cookie", pre=True)
+    table_id: int = Field(default=0)
```

Comment:
Same comments regarding the `Field` usages below. Looks like most of them can be assigned directly without `Field(default)`. Can you verify this again?

Reply:
Changed this too.